zeroconf/__init__.py

```diff
@@ -483,15 +485,15 @@ def get_expiration_time(self, percent: float) -> float:
 
     def get_remaining_ttl(self, now):
         """Returns the remaining TTL in seconds."""
-        return max(0, (self.get_expiration_time(100) - now) / 1000.0)
+        return max(0, (self._expiration_time - now) / 1000.0)
 
     def is_expired(self, now: float) -> bool:
         """Returns true if this record has expired."""
-        return self.get_expiration_time(100) <= now
+        return self._expiration_time <= now
 
     def is_stale(self, now):
         """Returns true if this record is at least half way expired."""
-        return self.get_expiration_time(50) <= now
```

Comment:
To get some statistics on this change, I did an experiment. I have a network with 170 devices running Zeroconf. Then we were listening for Zeroconf devices on that network. To get a more realistic test, the Zeroconf ServiceBrowser is ran on a separate thread from a main thread. On the main thread an I/O limited call to QNetworkManager is made every 2 seconds in order to include performance penalties due to thread switching. The experiment was ran on a MacBook Air 2015 (Intel Core i7-5650U) through Ubuntu 19.10 and Python 3.7. I left this running for exactly one hour, both before and after this commit.

Before this commit, there were 132107499 (132M) calls to the `get_expiration_time` function, totalling 141.647s (just over 2 minutes).

After this commit, there were 1661203 (1.6M) calls to the `get_expiration_time` function, totalling 2.068s. This saved about 2 minutes of processing time out of the total 60 minutes, on average 3.88% processing power on my CPU.